test/integration/config-sftp.yml

```diff
@@ -11,8 +11,9 @@ clickhouse:
 sftp:
   address: "sshd"
   username: "root"
-  password: ""
-  key: "/id_rsa"
+  password: "JFzMHfVpvTgEd74XXPq6wARA2Qg3AutJ"
```

Comment:
could you add separatelly config for `config-sftp-auth-key.yaml` and `config-sftp-auth-password.yaml`

Reply:
Sure.

Reply:
Done